src/components/Sender.tsx

```diff
@@ -68,7 +69,8 @@ const useStyles = makeStyles((theme) => ({
 const Sender = () => {
   const classes = useStyles();
   const [chains, setChains] = useState<Array<string[]>>([]);
-  const { account, accounts, derivedAccount, setCurrentAccount } = useAccounts();
+  const { accounts, setCurrentAccount } = useAccounts();
+  const { account, companionAccount, senderAccountBalance, senderCompanionAccountBalance } = useAccountContext();
```

Comment:
It's a bit confusing to me to see two different hooks regarding the seemingly the same data. Like why do we `setCurrentAccount` using `useAccounts` but retrieve the `account` (assuming "current") with the other hook?

Reply:
you are right saying that are two hooks, but the `useAccountContext` is just pulling the account context only. In our project all hooks having the convention: `use<concern>Context()` is just pulling the context . 
What it is confusing is that `accounts` are read from the `useAccounts` hook instead of the context. I have changed that.